source/Cosmos.Core/ObjUtilities.cs

```diff
@@ -0,0 +1,19 @@
+﻿using IL2CPU.API.Attribs;
+using System;
+
+namespace Cosmos.Core
+{
+    public static unsafe class ObjUtilities
+    {
+        public static uint GetPointer(Delegate aVal)
+        {
+            return (uint)aVal.GetHashCode();
+        }
+
+        [PlugMethod(PlugRequired = true)]
+        public static uint GetPointer(Object aVal) { return 0; }
```

Comment:
We already have https://github.com/CosmosOS/Cosmos/blob/f7fd5696a37c146bc225573c014bf8caf7917fbf/source/Cosmos.Core/GCImplementation.cs#L108 which does more or less the same?
If you need a uint we also have https://github.com/CosmosOS/Cosmos/blob/f7fd5696a37c146bc225573c014bf8caf7917fbf/source/Cosmos.Core/GCImplementation.cs#L115